packages/build/src/npm-packages/publish.ts

```diff
@@ -33,6 +33,7 @@ export function publishNpmPackages(
       [
         'publish',
         'from-package',
+        '--no-private',
```

Comment:
Not sure if this changes behavior in a meaningful way compared to what’s happening right now, but this is definitely what we _want_ to happen